test/sanitize-html.test.js

```diff
@@ -105,19 +106,19 @@ describe('Sanitize HTML', () => {
 
   test('<iframe> tag to a spotify embed should not get removed', () => {
     const data = sanitizeHTML(
-      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https%3A%2F%2Fmedium.com%2F%40pedrofonsecadev%2Ftelescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
```

Comment:
is this what we get from the parser?

Reply:
Not the symbols, those should be what comes out after sanitization, not prior. I can fix that